elements/lisk-cryptography/src/keys.ts

```diff
@@ -81,3 +85,98 @@ export const getAddressFromPrivateKey = (privateKey: string): string => {
 
 	return getAddressFromPublicKey(publicKey);
 };
+
+const GENERATOR = [0x3b6a57b2, 0x26508e6d, 0x1ea119fa, 0x3d4233dd, 0x2a1462b3];
+
+// See for details: https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md
+const polymod = (uint5Array: number[]): number => {
+	let chk = 1;
+	for (const value of uint5Array) {
+		// eslint-disable-next-line no-bitwise
+		const top = chk >> 25;
```

Comment:
Please add comments explaining the statement, will be very helpful when we read the code in a month or two. 

> Get 25 bits from the right side

Reply:
I feel that the above comment link is sufficient.